onnxruntime/contrib_ops/cpu/bert/attention_base.h

```diff
@@ -26,8 +26,9 @@ class AttentionBase {
                      int sequence_length,
                      int& past_sequence_length) const;
 
-  int num_heads_;           // number of attention heads
-  bool is_unidirectional_;  // whether every token can only attend to previous tokens.
+  int num_heads_;             // number of attention heads
+  mutable int head_size_;     // size of each attention head
```

Comment:
Does this keyword necessary?

Reply:
Yes, it needs to be updated if the heads are pruned.